DNN Platform/Website/admin/Modules/Export.ascx.cs

```diff
@@ -134,9 +134,16 @@ private string ExportModule(int moduleID, string fileName, IFolderInfo folder)
                             strMessage = Localization.GetString("ExportNotSupported", LocalResourceFile);
                         }
                     }
-                    catch
+                    catch (Exception ex)
```

Comment:
Ideally we should capture a more specific error

Reply:
Would you catch specifics and then the generic exception? Because the previous behaviour just catched anything and showed a generic error message.

Reply:
> 
> 
> Ideally we should capture a more specific error

You'd like me to rewrite this using catch (FooException ex) for each exception type instead of the if-then?

Reply:
FWIW, I'm happy with this as-is

Reply:
Ideally I'd like to see an exception filter or otherwise used, but I'll let it go